README.md

```diff
@@ -1,40 +1,55 @@
-# Openssl
+# OpenSSL
 
 [![Build Status](https://travis-ci.org/ruby/openssl.svg?branch=master)](https://travis-ci.org/ruby/openssl)
 
-OpenSSL provides SSL, TLS and general purpose cryptography.  It wraps the OpenSSL library.
+OpenSSL provides SSL, TLS and general purpose cryptography. It wraps the
+OpenSSL library.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+The openssl gem is available at [rubygems.org](https://rubygems.org/gems/openssl).
```

Comment:
`https://rubygems.org/gems/openssl` is already used another user. I will contact rubygems.org admin and openssl  gem owner for transfer namespace to us